stdlib/public/SwiftShims/Visibility.h

```diff
@@ -59,11 +59,16 @@
 # define SWIFT_RUNTIME_EXPORT __attribute__((__visibility__("default")))
 #endif
 
-#elif __CYGWIN__
-# define SWIFT_RUNTIME_EXPORT 
 #else
-// __dllexport/__dllimport for Windows?
-# error "Unimplemented object format"
+# if defined(__CYGWIN__)
```

Comment:
As mentioned in the commit message, there is no preprocessing macro to identify COFF.  '__MACH__' identifies MachO, '__ELF__' identifies ELF.  Since LLVM only supports those three, make neither of them assume COFF.